src/icon-manifest/supportedFolders.ts

```diff
@@ -43,7 +43,7 @@ export const extensions: IFolderCollection = {
     { icon: 'include', extensions: ['include', 'includes', 'incl'], format: FileFormat.svg },
     { icon: 'less', extensions: ['less', '_less'], format: FileFormat.svg },
     { icon: 'library', extensions: ['lib', '.lib'], format: FileFormat.svg },
-    { icon: 'locale', extensions: ['locale', 'locales', 'i18n', 'g11n'], format: FileFormat.svg },
+    { icon: 'locale', extensions: ['lang', 'language', 'locale', 'locales', 'i18n', 'g11n'], format: FileFormat.svg },
```

Comment:
@jens1o, what about `languages`?

Reply:
I didn't saw that ever in the wild. Do you have an example for it?

Reply:
Not really, but I see it more likely that someone names their language folder with `languages` instead of the singular name. Usually, one tends to have more than one language and I'm just following the same logic we applied with `locale` and `locales`. If I were to name a folder with languages I would definetely use the plural.

Reply:
Point